js/tinymce/classes/dom/EventUtils.js

```diff
@@ -138,15 +138,15 @@ define("tinymce/dom/EventUtils", [], function() {
 	function bindOnReady(win, callback, eventUtils) {
 		var doc = win.document, event = {type: 'ready'};
 
-		if (eventUtils.domLoaded) {
+		if (eventUtils.isDomLoaded()) {
 			callback(event);
 			return;
 		}
 
 		// Gets called when the DOM is ready
 		function readyHandler() {
-			if (!eventUtils.domLoaded) {
```

Comment:
I'm not sure if it matters, but I think there is a subtle change in logic here where when it was `!eventUtils.domLoaded` this statement could only be true once, whereas now, because we check the state each time and don't necessarily change the value, it could be true more than once. I'm not aware of any problems this might cause, but I think it means in theory that the callback could be called more than once.